code/__DEFINES/traits.dm

```diff
@@ -628,6 +628,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_NIGHT_VISION "night_vision"
 #define TRAIT_LIGHT_STEP "light_step"
 #define TRAIT_SPIRITUAL "spiritual"
+#define TRAIT_CHRONIC_ILLNESS "chronic_illness"
```

Comment:
why did you add this if it does nothing

Reply:
Trying to find a way to fix bug 1 I mentioned in the PR.